packages/valory/skills/termination_abci/behaviours.py

```diff
@@ -0,0 +1,522 @@
+        return tx_hash
+
+    def _get_multisend_tx(self) -> Generator[None, None, Optional[str]]:
+        """This method compiles a multisend transaction to give ownership of the safe contract to the service owner."""
+        transactions: List[Dict] = []
+        owner_to_be_swapped: Optional[str] = None
+        threshold = 1
+        safe_owners = yield from self._get_safe_owners()
+        if safe_owners is None:
+            return None
+
+        # we remove all but one safe owner
+        for owner in safe_owners:
+            if owner_to_be_swapped is None:
+                owner_to_be_swapped = owner
+                continue
+
+            # we generate a tx to remove the current owner
+            remove_tx = yield from self._get_remove_owner_tx(
+                owner_to_be_swapped, owner, threshold
+            )
+            if remove_tx is None:
+                return None
+
+            # we append it to the list of the multisend txs
+            transactions.append(
+                {
+                    "operation": MultiSendOperation.CALL,
+                    "to": self.synchronized_data.safe_contract_address,
+                    "value": _ETHER_VALUE,
+                    "data": HexBytes(remove_tx),
+                }
+            )
+
+        # we swap the last owner with the service owner
+        swap_tx = yield from self._get_swap_owner_tx(
+            owner_to_be_swapped,
+            owner_to_be_swapped,
+            self._service_owner_address,
+        )
+        if swap_tx is None:
+            return None
+
+        transactions.append(
+            {
+                "operation": MultiSendOperation.CALL,
+                "to": self.synchronized_data.safe_contract_address,
+                "value": _ETHER_VALUE,
+                "data": HexBytes(swap_tx),
+            }
+        )
+
+        response = yield from self.get_contract_api_response(
+            performative=ContractApiMessage.Performative.GET_RAW_TRANSACTION,  # type: ignore
+            contract_address=self.params.multisend_address,
+            contract_id=str(MultiSendContract.contract_id),
+            contract_callable="get_tx_data",
+            multi_send_txs=transactions,
+        )
+        if response.performative != ContractApiMessage.Performative.RAW_MESSAGE:
+            self.context.logger.error(
+                f"Couldn't compile the multisend tx. "
+                f"Expected response performative {ContractApiMessage.Performative.RAW_MESSAGE.value}, "
+                f"received {response.performative.value}."
+            )
+            return None
+
+        multisend_data = cast(str, response.raw_transaction.body["data"])[2:]
+        return multisend_data
+
+    def wait_for_termination_majority(self) -> Generator:
+        """
+        Wait until we reach majority on the termination transaction.
+        :yield: None
+        """
+        yield from self.wait_for_condition(self._is_termination_majority)
```

Comment:
so what happens without timeout. We wait, if we do not get majority the RESET_TIMEOUT will kick in and the system transitions?

Reply:
This round is immune to transitions.